internal/k8s/controller.go

```diff
@@ -2227,58 +2164,84 @@ func (lbc *LoadBalancerController) getPolicies(policies []conf_v1.PolicyReferenc
 	return result, errors
 }
 
-func (lbc *LoadBalancerController) addJWTSecrets(policies []*conf_v1alpha1.Policy, jwtKeys map[string]*api_v1.Secret) error {
+func (lbc *LoadBalancerController) addJWTSecrets(secrets map[string]*configs.SecretReference, policies []*conf_v1alpha1.Policy) error {
```

Comment:
Should we call this var `secretRefs` so the reader can assume it's type to be `map[string]*configs.SecretReference` instead of `map[string]*api_v1.Secret` without reading the func signature? 

```suggestion
func (lbc *LoadBalancerController) addJWTSecrets(secretRefs map[string]*configs.SecretReference, policies []*conf_v1alpha1.Policy) error {
```

Reply:
renamed